replication-api-impl/pom.xml

```diff
@@ -280,14 +312,25 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Import-Package>
                             !org.jvnet.staxex,
+                            !io.swagger.annotations,
                             ddf.security.permission,
                             ddf.security.encryption;version="[1.0.0, 3.0.0]",
                             *
                         </Import-Package>
                         <Export-Package>
                             org.codice.ditto.replication.api.impl.data,
                             org.codice.ditto.replication.api.impl.mcard,
-                            org.codice.ditto.replication.api.impl.modern
+                            org.codice.ditto.replication.api.impl.modern,
+
+                            <!-- cxf discovery client -->
+                            com.connexta.discovery.rest.cxf.client,
```

Comment:
I know there was some class path issues going on but this still doesn't seem like the right thing to do.